src/controllers/document_upload/document.upload.controller.ts

```diff
@@ -157,10 +159,10 @@ const getUploadFinishedCallback = (req: Request,
 
       // TODO OBJ-101 - file type error - uncomment below should do it
```

Comment:
Guessing that TODO comment should now be deleted if this is the full and final implementation of OBJ-101.

Reply:
Removed the TODO